doc/nrf/protocols/thread/sed_ssed.rst

```diff
@@ -55,7 +55,15 @@ Comparison of SED and SSED
 **************************
 
 Compared to an SED, an SSED has no drawbacks for transmission and provides reduced power consumption and network traffic.
-This means you should configure your SED devices as SSEDs whenever possible.
+
+Based on the SED and SSED characteristics, use SSED for cases that meet the following criteria:
+
+* You need a small latency (small CSL interval).
+  As a rule of thumb, use SSED when the application requires to poll more often than every 20 seconds, but make sure to make detailed measurements for your use case.
+* You want to have similar or smaller power consumption than with SED.
```

Comment:
From what I understand, SSEDs are viable to use only when the added code and complexity are justified by the lower power usage. Frequent polling (eg. at least every 20s as proposed by Eduardo) makes the device a candidate to be a viable SSED, given the device and parent uncertainty (among other parameters) is good enough - so the decision whether to use SSED still requires some application-specific data.
We could write something along those lines - the more frequent polling, the better candidate the device is to be SSED, as long as other parameters don't make the transmission window's size abysmal.
The important thing here is also that parent devices from other vendors might have big uncertainty, which will result in a big transmission window regardless of how low the uncertainty is on the child (SSED).

Reply:
@LuDuda , any opinions about Piotr's proposal?

Reply:
Good suggestions from Piotr. In general I'm a bit afraid to "recommend" SSED role blindly for manufacturers even if latency is less than ~20s. 

The whole Thread/Matter ecosystem is not yet fully ready. I would say that the default option is SED. However, SSED is great when it's parent has small CSL uncertainty as well accuracy. We don't have an example in SDK, but customer may use information about parent's CSL parameters (uncertainty and accuracy) to switch to SSED role or not. This can be done just after attachment dynamically.

In general what Piotr suggested is great. E.g. that SSED is viable option and may decrease power consumption in case the CSL interval is small (~<20s) as well as parent has decent CSL parameters.